src/timeout.c

```diff
@@ -74,6 +81,11 @@ int clientsCronHandleTimeout(client *c, mstime_t now_ms) {
                 unblockClient(c);
         }
     }
+    if (drop_client) {
+        serverLog(LL_VERBOSE,"Closing idle client");
```

Comment:
since we re-place it, so i suppose we can add the space, your call
```suggestion
        serverLog(LL_VERBOSE, "Closing idle client");
```